packages/cart/src/models/shipping-method.ts

```diff
@@ -16,16 +16,20 @@ import ShippingMethodAdjustmentLine from "./shipping-method-adjustment-line"
 import ShippingMethodTaxLine from "./shipping-method-tax-line"
 
 @Entity({ tableName: "cart_shipping_method" })
+@Check<ShippingMethod>({ expression: (columns) => `${columns.amount} >= 0` })
 export default class ShippingMethod {
   @PrimaryKey({ columnType: "text" })
   id: string
 
+  @Property({ columnType: "text" })
+  cart_id: string
+
   @ManyToOne(() => Cart, {
     onDelete: "cascade",
     index: "IDX_shipping_method_cart_id",
-    fieldName: "cart_id",
+    nullable: true,
   })
-  cart: Cart
+  cart?: Cart | null
```

Comment:
```suggestion
  cart: Cart | null = null
```

Reply:
This is different from our convention. Why would you like to change this?

Reply:
because it make it naturally optional and it does not alllow for undefined value

Reply:
Cool, can I get you to update our conventions on this? 🙏

Reply:
sure, I ve mentionned it on slack too but nobody reacted to it 😂

Reply:
it would also simplify all our read interfaces, only the relations could be marked as `?`

Reply:
I ve updated the convention document